packages/core/src/core.ts

```diff
@@ -775,7 +775,7 @@ class ThresholdKey implements ITKey {
     return decrypt(toPrivKeyECC(this.privKey), encryptedMessage);
   }
 
-  async setTKeyStoreItem(moduleName: string, data: TkeyStoreItemType): Promise<void> {
+  async setTKeyStoreItem(moduleName: string, data: TkeyStoreItemType, sync?: boolean): Promise<void> {
```

Comment:
change this to updateMetadata?. same as other functions

Reply:
Updated.